tests/L0/run_optimizers/test_fused_novograd.py

```diff
@@ -0,0 +1,168 @@
+import torch
+from torch.optim import Optimizer
+import math
+import apex
+import unittest
+
+from .test_fused_optimizer import TestFusedOptimizer
```

Comment:
This import will fail with:
```python
ImportError: attempted relative import with no known parent package
```
Could you remove the dot at the beginning?

Reply:
Done.